component/cleanup-cronjob.libsonnet

```
@@ -0,0 +1,96 @@
+/*
+* Deploys the control-api API server
```

Should probably be something like

```suggestion
* Deploys the control-api billing entity cleanup cronjob
```